Projects/App/Sources/Screens/ChatScreen/Reusables/Message+Reusable.swift

```diff
@@ -369,6 +398,118 @@ extension Message: Reusable {
                         }
 
                         contentContainer.addArrangedSubview(imageViewContainer)
+                    case let .crossSell(url):
+                        let crossSaleMainContainer = UIView()
+                        crossSaleMainContainer.backgroundColor = .clear
+                        let crossSaleContainer = UIView()
+                        crossSaleContainer.backgroundColor = .clear
+                        let loadingIndicator = UIActivityIndicatorView(style: .medium)
+                        loadingIndicator.color = UIColor.brand(.chatMessage)
+                        loadingIndicator.hidesWhenStopped = true
+                        crossSaleMainContainer.addSubview(crossSaleContainer)
+                        crossSaleContainer.snp.makeConstraints { make in
+                            make.leading.trailing.top.bottom.equalToSuperview()
+                        }
+                        crossSaleMainContainer.addSubview(loadingIndicator)
+                        loadingIndicator.snp.makeConstraints { make in
+                            make.centerX.centerY.equalToSuperview()
+                        }
+                        crossSaleContainer.isUserInteractionEnabled = true
+                        let imageView = UIImageView()
+                        crossSaleContainer.addSubview(imageView)
+                        imageView.snp.makeConstraints { make in
+                            make.leading.equalToSuperview()
+                            make.trailing.equalToSuperview()
+                            make.top.equalToSuperview().offset(message.shouldShowTimeStamp ? 12 : 12)
+                            make.width.equalTo(imageView.snp.height)
+                        }
+
+                        imageView.layer.cornerRadius = 20
+                        imageView.clipsToBounds = true
+                        //title //subtitle
```

Comment:
why is this here?

Reply:
Thats something we did during maker days. Not a big thing :) but looks nice